data/multiplayer/scenarios/2p_Elensefar_Courtyard.cfg

```diff
@@ -96,7 +96,7 @@
             [/or]
             [or]
                 x,y=32-36,9-19
-                terrain=R*,Gg^Emf,Rp^Uf,Rp^Dr,Rd^Uf
+                terrain=R*,Gg^Emf,Tb^Tf,Rp^Dr
```

Comment:
This needs to be in #5145 , along with the change to the corresponding map.

Reply:
Done.